rules-tests/CodeQuality/Rector/Expression/TernaryFalseExpressionToIfRector/Fixture/some_class.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\Expression\TernaryFalseExpressionToIfRector\Fixture;
+
+final class SomeClass
+{
+    public function run($value, $someMethod)
+    {
+        $value ? $someMethod->call($value) : false;
```

Comment:
How about negation `! $value` ?

```
! $value ? false : execute();
! $value ? execute : false;
```

Reply:
I've added 2nd one... how would you handle:

```php
! $value ? false : execute();
```

Reply:
Yes, that should be skipped